packages/fluentui/docs/.eslintrc.json

```diff
@@ -3,7 +3,7 @@
   "overrides": [
     {
       // These files don't have actual dependencies
-      "files": ["src/examples/**/*.steps.ts", "src/examples/**/commonScreenerSteps.ts"],
+      "files": ["src/vr-tests/**/*.stories.tsx", "src/vr-tests/**/*.stories.ts", "src/vr-tests/**/*.ts"],
```

Comment:
Will create a follow-up PR removing all the `eslint-disable-next-line import/no-extraneous-dependencies` lines that were added to stories within the `vr-tests` subfolder

Reply:
#26054